synapse/models/orgs.py

```diff
@@ -485,6 +488,58 @@ def getModelDefs(self):
                     ('contest', ('ou:contest', {}), {
                         'doc': 'The contest that the person attended.',
                     }),
+                    ('preso', ('ou:preso', {}), {
+                        'doc': 'The presentation that the person attended.',
+                    }),
+                )),
+                ('ou:preso', {}, (
+
+                    ('organizer', ('ps:contact', {}), {
+                        'doc': 'Contact information for the primary organizer of the presentation.'}),
+
+                    ('sponsors', ('array', {'type': 'ps:contact'}), {
+                        'doc': 'An array of contacts which sponsored the presentation.'}),
+
+                    ('presenters', ('array', {'type': 'ps:contact'}), {
+                        'doc': 'An array of contacts which gave the presentation.'}),
+
+                    ('title', ('str', {'lower': True}), {
+                        'doc': 'The full name of the presentation.',
+                        'ex': 'Synapse 101 - 2021/06/22'}),
+
+                    ('desc', ('str', {'lower': True}), {
+                        'doc': 'A description of the presentation.',
+                        'disp': {'hint': 'text'}}),
+
+                    ('time', ('time', {}), {
+                        'doc': 'The presentation start time.'}),
+
+                    ('duration', ('duration', {}), {
+                        'doc': 'The duration of the presentation.'}),
+
+                    ('loc', ('loc', ()), {
+                        'doc': 'The geopolitical affiliation of the presentation.'}),
+
+                    ('place', ('geo:place', ()), {
+                        'doc': 'The geo:place node where the presentation was held.'}),
+
+                    ('deck:url', ('inet:url', ()), {
+                        'doc': 'The URL hosting a copy of the presentation materials.'}),
+
+                    ('deck:file', ('file:bytes', ()), {
+                        'doc': 'A file containing the presentation materials.'}),
+
+                    ('attendee:url', ('inet:url', ()), {
+                        'doc': 'The URL visited by live attendees of the presentation.'}),
+
+                    ('recording:url', ('inet:url', ()), {
+                        'doc': 'The URL hosting a recording of the presentation.'}),
+
+                    ('recording:file', ('file:bytes', ()), {
+                        'doc': 'A file containing a recording of the presentation.'}),
+
+                    ('conference', ('ou:conference', ()), {
+                        'doc': 'The conference which hosted the presentation.'}),
```

Comment:
Nitpicky and largely for the sake of argument, but if a preso can be given at a conference, could a preso also be given at an ou:conference:event or an ou:meet? Add now for completeness or add later when need arises?

Reply:
Leaving that for future decisions :)